### ISSUE TITLE
Problem connecting to OPC server without a certificate

### ISSUE THREAD

**Issue by temmat** [biancode/node-red-iiot-opcua-publicbeta#4]
Hello,
it seems there is a problem with connecting to an OPC server that does not provide a certificate (seems to be something similar to the issue reported in the node-opcua github here:
https://github.com/node-opcua/node-opcua/issues/67

An AssertionError: endpoint.serverCertificate is reportes in the node-red terminal when trying to use  the node-red-contrib-iiot-opcua Browser (even using the example browser flow and connecting to an OPC UA server without a certificate). 

UAExpert connects fine and it correctly report at connection time "The server returned no certificate, all certificate checks will be skipped.", however node-red-contrib-iiot-opcua fails. 

Since I'm able to connect correctly with node-red-contrib-opcua is it possible that the issue reported above was not fixed here ? (Since it seems there is a common background.)

**Comment by biancode** [comment]
the version 0.1.0-x of node-opcua has some [issues](https://github.com/node-opcua/node-opcua/issues/390) - I will test it, too.

**Comment by biancode** [comment]
You should report that issue to the [node-opcua library](https://github.com/node-opcua/node-opcua/issues), please. Maybe that is a known issue for @erossignon.

**Comment by biancode** [comment]
here please attach you log with [DEBUG](https://github.com/biancode/node-red-iiot-opcua-publicbeta/wiki/DEBUG) information for the client and the connector

**Comment by temmat** [comment]
Just in case I haven't stated it clearly enough: connecting with the node-red-contrib-opcua works, connecting with node-red-contrib-iiot-opcua does not; using the same connection settings and to the same server without the certificate and without login credentials (no certificate no username and password).

@erossignon has already commented on it in the issue that I provided a link to. As it seems a patch has also already been included in node-opcua library.  @bastervrugt in the same discussion thread, suggested a way of dealing with it and there is something similar included in my .node-red/node_modules/node-opcua-client/src/opcua_client.js file.

Specifically I'm talking about the code:
```javascript
// if server does not provide certificate use unencrypted password
    if (serverCertificate === null) {
    ...
```
Is it possible that node-red-contrib-iiot-opcua uses a fork or version of node-opcua library that doesn't have the fix included yet ? It was just a thought. Maybe it's another issue completely.

I will attach the debug logs as soon as I can.

**Comment by biancode** [comment]
node-red-contrib-opcua installs the old v0.0.61 of the node-opcua and the package here installs the very new node-opcua v0.1.0-12 with a whole new structure - I will test it, too.

**Comment by biancode** [comment]
found ... will be fixed at v1.0.15+

**Comment by biancode** [comment]
![image](https://user-images.githubusercontent.com/4369867/32633170-c27b39e2-c5a6-11e7-8292-a89c1c7201fd.png)
